docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -420,6 +422,18 @@ table description supplier are:
 * There is no need to define tables manually.
 * Some Protobuf specific types like ``oneof`` are supported and mapped to JSON.
 
+When using Protobuf decoder with the Confluent table description supplier, the
+following additional steps must be taken:
+
+- Copy the JARs ``kafka-protobuf-provider`` and ``kafka-protobuf-types`` for
+  Confluent version 7.3.1 to the Kafka connector plugin directory on all nodes in
+  the cluster.
+- The JARs are provided under the Confluent license by Confluent. Accept those
```

Comment:
This needs to be first and have details on how to get them from Confluent .. or link to their site or whatever

Reply:
Added link to first bullet point. https://github.com/trinodb/trino/pull/17924#discussion_r1232059940

The order is as recommended by Ashhar and makes sense to me.

Reply:
How do you accept the license? The download is wide open so there is not license to accept.

Reply:
You accept license by agreeing to use those things. It's a bit hostile but that's how it is.
The pom for each of them includes the license under which they are released (Confluent Community License).

Reply:
Can probably be reworded as "By copying those JARs and using them you agree to the terms of the license under which Confluent makes them available"

Reply:
maybe add a link to https://github.com/confluentinc/schema-registry/blob/master/LICENSE-ConfluentCommunity - which is where the JARs come from and that's their license.